src/utilities/BatchTable.js

```diff
@@ -71,4 +72,11 @@ export class BatchTable extends FeatureTable {
 
 	}
 
+	getPropertyArray( key, componentType = null, type = null ) {
```

Comment:
Do we need the `componentType` and `type` fields?

Reply:
I kept them since they were in `getData` but I was wondering the same thing. Personally I don't see any use cases. I will remove them.